documentation/milestones/individual_milestones/get_opportunities.md

```diff
@@ -13,7 +13,9 @@
 ## Short description
 <!-- Required -->
 
-Deploy a public API endpoint to PROD that allows users to see at least one field per grant opportunity listed in grants.gov
+**What:** Deploy a public API endpoint to PROD iteratively that allows users to see at least one field per grant opportunity listed in grants.gov. 
+**Why:** Build iteratively, validate connection to the database, and set the groundwork to continue to enhance the API.
+**Who:** This release of the API will be mainly focused on delivering for the API consumer
```

Comment:
Markdown concatenates lines that don't have spaces between them, I added an extra space so each word will show up on a new line. Feel free to use bullets instead though

```suggestion
**What:** Deploy a public API endpoint to PROD iteratively that allows users to see at least one field per grant opportunity listed in grants.gov. 

**Why:** Build iteratively, validate connection to the database, and set the groundwork to continue to enhance the API.

**Who:** This release of the API will be mainly focused on delivering for the API consumer
```